### PR TITLE
Remove vendor dependencies and cleanup scripts.

### PR DESCRIPTION
**Pull request by jorgecuesta**
## Summary

This pull request involves a significant change to our project's dependencies. We have used forked versions of the `cosmjs` and `subql` packages. 

The primary reason behind opting for these forks was to address the compatibility issues with Cosmos SDK version `0.50.x` . The official maintainers have since [resolved](https://github.com/subquery/subql-cosmos/releases/tag/node-cosmos/4.0.0) these issues.

As such, I have removed the forked `cosmjs` and `subql` packages and replaced them with the official ones. This change aims to ensure our project remains current, reliable, and secure. Additionally, using the official packages enables us to take advantage of future improvements and bug fixes more efficiently.

I have thoroughly tested these changes and confirmed that they don't introduce any adverse effects on our project's functionality.

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [ ] Bug fix
- [x] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

**Documentation changes** (only if making doc changes)
- [ ] `make docusaurus_start`; only needed if you make doc changes

**Local Testing** (only if making code changes)
- [ ] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- See [quickstart guide](https://dev.poktroll.com/developer_guide/quickstart) for instructions

**PR Testing** (only if making code changes)
- [ ] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR.
    - **THIS IS VERY EXPENSIVE**, so only do it after all the reviews are complete.
    - Optionally run `make trigger_ci` if you want to re-trigger tests without any code changes
    - If tests fail, try re-running failed tests only using the GitHub UI as shown [here](https://github.com/pokt-network/poktroll/assets/1892194/607984e9-0615-4569-9452-4c730190c1d2)

## Sanity Checklist

- [x] I have tested my changes using the available tooling
- [x] I have commented my code
- [x] I have performed a self-review of my own code; both comments & source code
- [ ] I create and reference any new tickets, if applicable
- [ ] I have left TODOs throughout the codebase, if applicable

### CODE REVIEW

File: docker/node.dockerfile [hunk 1]
```diff
@@ -82,18 +77,16 @@ COPY --from=builder /app/project.ts /app/schema.graphql /app/tsconfig.json /home
...
 
-# TODO_MAINNET(@bryanchriswhite): Add the .gmrc once migrations are available.
-#COPY ./.gmrc /app/.gmrc
-
 # Allow execution for every shell script at scripts folder
 RUN find /home/app/scripts -type f -name "*.sh" -exec chmod +x {} \;
 
 # Set user as app
 USER app
+# Required after this version https://github.com/subquery/subql-cosmos/releases/tag/node-cosmos/4.0.0
```
Comment (Olshansk) [inline docker/node.dockerfile:89]: Great comment!

File: README.md [hunk 2]
```diff
@@ -18,10 +18,9 @@ To learn more about SubQuery, [see their docs](https://academy.subquery.network)
   - [Explore via postgres](#explore-via-postgres)
 - [Getting Started](#getting-started)
   - [tl;dr local development (if not your first time)](#tldr-local-development-if-not-your-first-time)
-  - [1. Ensure submodules are updated](#1-ensure-submodules-are-updated)
-  - [2. Install dependencies](#2-install-dependencies)
-  - [3. Generate types](#3-generate-types)
-  - [4. Run](#4-run)
+  - [1. Install dependencies](#1-install-dependencies)
```
Comment (Olshansk) [inline README.md:21]: @jorgecuesta FYI - I pushed these changes

**Review by Olshansk — APPROVED** [review]
@jorgecuesta LGTM but please make sure the CI passes before merging it in!

**Comment by jorgecuesta** [comment]
> @jorgecuesta LGTM but please make sure the CI passes before merging it in!

Fixed

**Review by Olshansk — APPROVED** [review]

**Pull request merged**